ansible_collections/arista/avd/roles/eos_designs/templates/network_services/main.j2

```diff
@@ -12,6 +12,13 @@
 
 {% endif %}
 
+{% if switch.vtep is arista.avd.defined(true) or
```

Comment:
Does this add anything that the stuff at 56/63 doesn't already cover except the "set tenants = dict()"?

Reply:
it was a miss from rebase. I will remove this. Thank you.

Reply:
Fixed